modules/gapi/samples/face_beautification.cpp

```diff
@@ -4,6 +4,12 @@
 //
 // Copyright (C) 2018-2019 Intel Corporation
 
+#if defined(HAVE_OPENCV_GAPI) && defined(HAVE_INF_ENGINE)
+
+#if INF_ENGINE_RELEASE <= 2018050000
+#   error G-API IE module supports only OpenVINO IE >= 2019 R1
+#endif
```

Comment:
You don't need it here. G-API's public API doesn't depend on either version of OpenVINO. This is an internal check and the library won't build itself because of this.

Reply:
Yes, it is true, thank you